packages/entities/entities-shared/src/components/entity-empty-state/EntityEmptyState.vue

```diff
@@ -188,9 +191,18 @@ const { i18n: { t } } = composables.useI18n()
   }
 
   .entity-empty-state-card-container {
-    display: grid !important;
+    $feature-card-width: 312px;
```

Comment:
question: why do you set fixed width for cards?

Reply:
This (setting card width) is the only way I could figure out how to meet all three of these requirements:

- page content is center aligned
- card view can only display 2 columns max
- if only one column has content, it should be center aligned rather than left aligned

Definitely open to suggestions.

![Uploading image.png…]()